packages/dx-react-scheduler/src/plugins/scheduler-core.tsx

```diff
@@ -4,6 +4,9 @@ import {
 } from '@devexpress/dx-react-core';
 import { appointments, formatDateTimeGetter } from '@devexpress/dx-scheduler-core';
 import { SchedulerProps } from '../types';
+import { memoize } from '@devexpress/dx-core';
+
+const memoFormatDateTimeGetter = memoize(locale => formatDateTimeGetter(locale));
```

Comment:
Should we use `memo` here?

Reply:
We need to use `memo` here in order to be able to memoize cells.

Reply:
Can we change the `formatDateTimeGetter` function code without creating a new function? I mean, that we should use memo inside `scheduler-core` package.